database_schemas/cassandra/benchflow.cql

```diff
@@ -135,7 +139,7 @@ CREATE TABLE container_properties (
 );
 
 CREATE TABLE host_properties (
-  host_id uuid,
+  host_id text,
```

Comment:
@VincenzoFerme Performance remains the same, we are not using the id as the partition key either way.